### PR TITLE
query node - nft collectors

### PR DESCRIPTION
**Pull request by ondratra**
This PR adds support to the biggest NFT collectors list. Solves https://github.com/Joystream/joystream/issues/3301

The biggest NFT collectors can be queried like this:
```
channelNftCollectors(where: {channel: {id_eq: 3}}, orderBy: [amount_DESC, lastIncreaseAt_ASC]) {id, channel {id}, member {id}, amount, lastIncreaseAt }
```

### CODE REVIEW

**Review by zeeshanakram3 — APPROVED** [review]
LGTM.

File: query-node/schemas/content.graphql [hunk 1]
```diff
@@ -72,6 +72,9 @@ type Channel @entity {
 
   "List of channel collaborators (members)"
   collaborators: [Membership!]
+
+  #"List of biggest video NFT owners."
+  #nftCollectors: [Membership!]!
```
Comment (kdembler) [inline query-node/schemas/content.graphql]: Should this be commented out?

File: query-node/mappings/src/derivedPropertiesManager/applications/videoNftCollectors.ts [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+}
+
+interface IOwnerType {
+  member: Membership | undefined
+  curatorGroup: CuratorGroup | undefined
+}
+
+class NftCollectorsExecutor implements IExecutor<OwnedNft, IOwnedNftChange, ChannelNftCollectors> {
+  async loadDerivedEntities(store: DatabaseManager, ownedNft: OwnedNft): Promise<ChannelNftCollectors[]> {
+    // TODO: find way to reliably decide if channel, etc. are loaded and throw error if not
```
Comment (kdembler) [inline query-node/mappings/src/derivedPropertiesManager/applications/videoNftCollectors.ts:58]: Do we want to resolve this before merging?

File: query-node/schemas/membership.graphql [hunk 3]
```diff
@@ -119,6 +119,8 @@ type Membership @entity {
 
   "NFTs owned by this member"
   ownedNfts: [OwnedNft!]! @derivedFrom(field: "ownerMember")
+
+  #nftCollectorInChannels: [Channel!]! @derivedFrom(field: "nftCollectors")
```
Comment (kdembler) [inline query-node/schemas/membership.graphql]: Should this be commented out?

File: query-node/schemas/content.graphql (hunk 1 above)
Comment (ondratra) [reply query-node/schemas/content.graphql]: These commented lines have been removed, and proper relationship has been created here.

File: query-node/schemas/membership.graphql (hunk 3 above)
Comment (ondratra) [reply query-node/schemas/membership.graphql]: These commented lines have been removed, and proper relationship has been created here.

File: query-node/mappings/src/derivedPropertiesManager/applications/videoNftCollectors.ts (hunk 2 above)
Comment (ondratra) [reply query-node/mappings/src/derivedPropertiesManager/applications/videoNftCollectors.ts:58]: No, I will fix this as a separate issue later on (either with an active video counter fix or later when implementing forum post counters).

**Pull request merged**